shared/fs/top-bar/loading.tsx

```diff
@@ -10,14 +11,28 @@ type OwnProps = {
 
 const mapStateToProps = (state, {path}: OwnProps) => ({
   _loadingPaths: state.fs.loadingPaths,
+  _pathItem: state.fs.pathItems.get(path, Constants.unknownPathItem),
 })
 
 const emptySet = I.Set()
 
-const mergeProps = (stateProps, dispatchProps, {path}: OwnProps) => ({
-  loading: stateProps._loadingPaths.get(path, emptySet).size > 0,
-})
+const mergeProps = (stateProps, dispatchProps, {path}: OwnProps) => {
+  // Only show the loading spinner when we are first-time loading a pathItem.
+  // If we already have content to show, just don't show spinner anymore even
+  // if we are loading.
+  if (stateProps._pathItem.type === Types.PathType.Unknown) {
```

Comment:
Looks like this is true for `private`,`public`, and `team`, the spinner is always on for those

Reply:
Opops good catch!